src/index.html

```diff
@@ -17,16 +17,33 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+      method="get"
+    >
+      <input
+        class="search search--big"
```

Comment:
The `class` attribute value `search search--big` seems a bit confusing. Typically, the `--` notation is used in BEM methodology to denote a modifier of a block or element. But here, `search` is repeated twice. If `search` is a block, and `big` is its modifier, it would be more appropriate to name the class `search search--big`.